Translated/Quests/Main Story/ms0010100104.csv

```diff
@@ -0,0 +1,13 @@
+Title#0,"""Oranje's Advice"""
+Summary#0,"""次に、オペレーターのオランジェから、<br>フードスタンドの利用方法を教わる。<br><br>“クイックフード”を作るために、<br>まず、セントラルシティの周辺に棲息する<br>テイムズから肉を入手する。"""
+text0101#0,"""Talk to Oranje"""
+text0102#0,"""Gather Tames Meat"""
+text0103#0,"""Slay Dian to obtain シャキリッタ肉"""
+text0104#0,"""Slay Sunny to obtain マタリーナ肉"""
+text0105#0,"""Slay Patt to obtain サッパリーナ肉"""
+text0106#0,"""Slay Moobel to obtain ガッツリーナ肉"""
```

Comment:
We have names for the meat now. In order, these will be "Sharp Meat", "Nourishing Meat", "Refreshing Meat" and "Healthy Meat". I don't know why they're slightly different here from the actual item names.

Reply:
Should i change "Slay" to "Subdue" ?